added file_fetcher and requirment.txt

memuzdak opened the pull request.

memuzdak committed:
added file_fetcher and requirment.txt

muhammadali286 reviewed (changes requested):
Lets handle all date related operations with `datetime` library. 
Lets remove comments, code should be self explanatory. 
Lets create separate modules for reading data, calculations and generating reports. 
Lets remove `requirement.txt` file. 
Lets follow `pep-8`

memuzdak committed:
Delete file_fetcher.py 

This file has all the code, but I was asked to implement the modular approach. So that's why I delete this file

memuzdak committed:
Delete requirement.txt file

I was asked by my trainer to remove it.
